BOJ/BOJ_1916.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from heapq import heappop, heappush
+input = sys.stdin.readline
+
+global graph
+
+def dijkstra(s, e):
+  q = [(0, s)]
+  distance = [float('inf')] * len(graph)
+  distance[s] = 0
+
+  while q:
+    dist, now = heappop(q)
+    if distance[now] < dist:
+      continue
+    for i in graph[now]:
```

Comment:
이 부분도 아래처럼 하고자 하는 목적에 맞도록 변수명을 하면 좋을 것 같아요.
왜냐하면 지금은 간단히 최단거리를 찾는 코드지만 나중에 카카오처럼 복잡한 구현이 첨가되면 이 리스트안에 들은게 뭐더라? 하는 시간이 엄청 잡아먹게 될거에요!
```suggestion
    for next_node, dist in graph[now]:
```